tokio-postgres/src/config.rs

```diff
@@ -50,6 +50,16 @@ pub enum SslMode {
     Require,
 }
 
+/// TLS negotiation configuration
```

Comment:
Consider linking to https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-SSLNEGOTIATION